src/components/Progress/sgds-progress.ts

```diff
@@ -0,0 +1,35 @@
+import { property } from "lit/decorators.js";
+import { classMap } from "lit/directives/class-map.js";
+import SgdsElement from "../../base/sgds-element";
+import styles from "./sgds-progress.scss";
+import { html } from "lit";
+
+/**
+ * @summary Provide up-to-date feedback on the progress of a workflow or action with simple yet flexible progress bars.
+ * @slot default - slot for progress-bar
+ *
+ * @cssprop --sgds-progress-height - Sets the height of the progress wrapper.
+ */
+
+export class SgdsProgress extends SgdsElement {
+  static styles = [SgdsElement.styles, styles];
+
+  /** Forwarded to the base wrapper of sgds-progress. Can be used to insert any utility classes such as `me-auto` */
+  @property({ type: String, reflect: true }) progressClasses: string;
+
+  render() {
+    return html`
+      <div
+        part="base"
```

Comment:
this csspart not documented. if not needed can remove?

Reply:
resolved in [a9b34cc](https://github.com/GovTechSG/sgds-web-component/pull/110/commits/a9b34ccc04c97965cc602a23aac0a06bce3fbb86)